examples/symfony-demo/zbuild.yml

```diff
@@ -21,10 +21,18 @@ sources:
 
 stateful_dirs:
   - var/session/
+  - data/
 
 integrations:
   - blackfire
 
+post_install:
+  - echo '' > .env
+  # @TODO: let zbuild create data/ folder with the right owner/group
+  - mkdir -p var/cache/ data/
+  # @TODO: find a proper way to warmup symfony cache
```

Comment:
use composer scripts ?

Reply:
They won't work for this example because Doctrine tries to connect to the DB to know its version :confused:

Reply:
rah still this issue :(